setup.py

```diff
@@ -56,14 +56,16 @@
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ],
 
+    # Require Python version equal or more than Python 3.
+    python_requires='>=3',
```

Comment:
why did we drop it?

Reply:
Python 2.7 handles dependencies poorly. So, it turned out the SDK was not working with 2.7 for last 2 weeks... However, no one ever reported the bug or complained. Moreover, some users were pushing for switch to Python 3 only.